fixtures/vitest-pool-workers-examples/misc/vitest.config.ts

```diff
@@ -11,9 +11,15 @@ export default defineWorkersProject({
 			workers: {
 				singleWorker: true,
 				miniflare: {
+					kvNamespaces: ["KV_NAMESPACE"],
 					outboundService(request) {
 						return new Response(`fallthrough:${request.method} ${request.url}`);
 					},
+					serviceBindings: {
+						ASSETS(request) {
+							return new Response(`assets:${request.method} ${request.url}`);
+						},
+					},
```

Comment:
How do we expect people to implement this? Is it worth calling out https://github.com/cloudflare/workers-sdk/blob/8ab8187f0426e1ea4ac8aa63bfde239b51347e74/packages/wrangler/src/miniflare-cli/assets.ts#L28  (what Wrangler uses to generate this) here?

Reply:
This is really a set of low level tests that are not exactly examples of how to use the integration.
The better example for users is the `pages-functions-unit-integration-self` fixture below.
That uses the `buildPagesASSETSBinding()` helper from `@cloudflare/vitest-pool-workers/config`.

Reply:
That being said, that helper method is not documented at all in this PR.
I'll create a follow-up ticket to add docs for that.

Reply:
#5909